docs/csharp/language-reference/compiler-messages/cs0050.md

```diff
@@ -1,35 +1,37 @@
 ---
 title: "Compiler Error CS0050"
 ms.date: 07/20/2015
-f1_keywords: 
+f1_keywords:
   - "CS0050"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS0050"
 ms.assetid: dead2d28-f4db-4afe-b8dd-38968625f7c3
 ---
 # Compiler Error CS0050
-Inconsistent accessibility: return type 'type' is less accessible than method 'method'  
-  
- The return type and each of the types referenced in the formal parameter list of a method must be at least as accessible as the method itself. For more information, see [Access Modifiers](../../programming-guide/classes-and-structs/access-modifiers.md).  
-  
-## Example  
- The following sample generates CS0050 because no accessibility modifier is supplied for `MyClass` and its accessibility therefore defaults to `private`.  
-  
-```csharp  
-// CS0050.cs  
-class MyClass //accessibility defaults to private  
-// try the following line instead  
-// public class MyClass   
-{  
-}  
-  
-public class MyClass2  
-{  
-    public static MyClass MyMethod()   // CS0050  
-    {  
-        return new MyClass();  
-    }  
-  
-    public static void Main() { }  
-}  
+
+Inconsistent accessibility: return type 'type' is less accessible than method 'method'
+
+ The return type and each of the types referenced in the formal parameter list of a method must be at least as accessible as the method itself. For more information, see [Access Modifiers](../../programming-guide/classes-and-structs/access-modifiers.md).
+
+## Example
+
+ The following sample generates CS0050 because no accessibility modifier is supplied for `MyClass` and its accessibility therefore defaults to `private`:
```

Comment:
```suggestion
 The following sample generates CS0050 because no accessibility modifier is supplied for `MyClass`, and its accessibility therefore defaults to `private`:
```